lib/UI/components/footer.dart

```diff
@@ -6,15 +6,11 @@ double largeFont = 24;
 
 class Footer extends StatelessWidget {
   @override
-  Widget build(BuildContext context) {
-    return LayoutBuilder(builder: (context, constraints) {
-      if (constraints.maxWidth < 650) {
-        return FooterLayoutMobile();
-      } else {
-        return FooterLayoutDesktop();
-      }
-    });
-  }
+  Widget build(BuildContext context) => LayoutBuilder(
```

Comment:
This was me butting in with my functional obsession - happy to undo!

Reply:
I like it!